src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
```

Comment:
The constants 'cost', 'smallDiscount', and 'bigDiscount' are well-named and make the code more readable. However, consider giving a more descriptive name to 'cost', such as 'DAILY_RENTAL_RATE', to clarify its role in the calculation.

Reply:
<img width="793" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/a4eea3ec-f1f8-4f55-9f7e-54bb079ef52e">
better to fix it